firebase-functions/src/main/java/com/google/firebase/functions/FirebaseFunctions.java

```diff
@@ -75,13 +85,26 @@
   private String urlFormat = "https://%1$s-%2$s.cloudfunctions.net/%3$s";
 
   FirebaseFunctions(
-      Context context, String projectId, String region, ContextProvider contextProvider) {
+      Context context,
+      String projectId,
+      String region,
+      ContextProvider contextProvider,
+      @Nullable EmulatedServiceSettings emulatorSettings) {
```

Comment:
Is there any value to holding this reference? Or is it only useful for this URL?

Reply:
Yeah we don't need it after we use it.